src/get-environments.ts

```diff
@@ -64,17 +60,12 @@ export async function getEnvironmentsForBranch() {
   return environments;
 }
 
-async function getEnvironments(organizationId: string) {
+async function getEnvironments() {
   try {
-    return apiClient.getEnvironments(organizationId);
+    return apiClient.getEnvironments();
   } catch (e) {
     console.log(e);
```

Comment:
where do console logs go in a vs code extension?

Reply:
19cf61ccf71d070dd6e6fcb73d4daaba85caa596
we don’t need this function anymore because we handle errors from getEnvironments [here]( 
https://github.com/env0/vscode-env0/blob/cd30acba8527c46c87ccd89b92ecdc4f95a5fa25/src/extension.ts#L55)

Reply:
I was just asking for general knowledge 👍🏻